README.md

```diff
@@ -474,14 +510,16 @@ total_failures = wavefront_sender.get_failure_count()
 
 1. Merge all the changes that need to go into the release to the master branch.
 2. Open the `setup.py` file from the top level directory of the project.
-3. Search for `version=` in the file to find the version number for example `1.8.10`.
-4. Log into Github, click **Releases** on the right, and click **Draft a new release**.
-5. For **Choose a tag**, choose the version you found in step 3, and prefix it with `v` for example `v1.8.10`. You need to enter the version where it says **Find or create new tag**.
+3. Search for `version=` in the file to find the version number for example `1.8.15`.
+4. Create a pull request, get it reviewed and approved, merge it after approval.
+5. Check [test.pypi.org](https://test.pypi.org/project/wavefront-sdk-python) for a published package, make sure it's production ready.
+6. Log into Github, click **Releases** on the right, and click **Draft a new release**.
```

Comment:
Log in to GitHub, click **Releases** on the right, and click **Draft a new release**.